third_party/move/move-compiler-v2/tests/op-equal/invalid2.move

```diff
@@ -0,0 +1,11 @@
+module 0x42::test {
+    fun test0() {
+        let x = false;
```

Comment:
What happens with code like:
```
    let x += 2;
```
?  Some examples, please.

Also, how about:
```
    let x = 3;
    let y = (x += 2) * (x -= 1);
```
and others like that?

Reply:
Done. See `invalid4.exp` and `invalid5.exp`. Assignment operator returns unit value rather than the updated value in Move.